frontend/src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router';
+import './Header.css'
+import logo from '../ING-logo-v2.jpg'
+
+export default class Header extends Component {
+    render() {
+        return (
+            <div className="header">
+                <Link to="/home" className="back">back</Link>
```

Comment:
https://en.bem.info/methodology/quick-start/ 

will give you some guidelines about styling.